README.md

```diff
@@ -1,2 +1,145 @@
+  * [Use MongoDB in a Request Handler](#use-mongodb-in-a-request-handler)
+  * [Working with Extended JSON](#working-with-extended-json)
+
+## Documentation
+### The latest documentation for the library is available [here](https://mongodb.github.io/mongodb-vapor/).
+
+### You can find a complete example project built using this library [here](https://github.com/mongodb/mongo-swift-driver/tree/main/Examples/VaporExample)!
+
+## Bugs / Feature Requests
+Think you've found a bug? Want to see a new feature in `mongodb-vapor`? Please open a case in our issue management tool, JIRA:
+
+1. Create an account and login: [jira.mongodb.org](https://jira.mongodb.org)
+2. Navigate to the SWIFT project: [jira.mongodb.org/browse/SWIFT](https://jira.mongodb.org/browse/SWIFT)
+3. Click **Create Issue** - Please provide as much information as possible about the issue and how to reproduce it.
+
+Bug reports in JIRA for all driver projects (i.e. NODE, PYTHON, CSHARP, JAVA) and the
+Core Server (i.e. SERVER) project are **public**.
+
+## Installation
+This library works with **Swift 5.2+** , and supports Linux and macOS usage. The minimum macOS version required is **10.15**.
+
+Installation is supported via [Swift Package Manager](https://swift.org/package-manager/).
+
+### Step 1: Install Required System Libraries (Linux Only)
+The driver vendors and wraps the MongoDB C driver (`libmongoc`), which depends on a number of external C libraries when built in Linux environments. As a result, these libraries must be installed on your system in order to build MongoSwift.
+
+To install those libraries, please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#prerequisites-for-libmongoc) from `libmongoc`'s documentation.
+
+### Step 2: Install the library
+Add this package along with Vapor as dependencies in your project's `Package.swift` file:
+
+```swift
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "VaporExample",
+    platforms: [
+        .macOS(.v10_15)
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "4.7.0")),
+        .package(url: "https://github.com/mongodb/mongodb-vapor", .upToNextMajor(from: "1.0.0"))
+    ],
+    targets: [
+        .target(
+            name: "App",
+            dependencies: [
+                .product(name: "Vapor", package: "vapor"),
+                .product(name: "MongoDBVapor", package: "mongodb-vapor")
+            ]
+        ),
+        .target(name: "Run", dependencies: [
+            .target(name: "App"),
+            .product(name: "MongoDBVapor", package: "mongodb-vapor")
+        ])
+    ]
+)
+```
+
+Then run `swift build` to download, compile, and link all your dependencies.
+
+## Example Usage
+### You can find a complete example project built using this library [here](https://github.com/mongodb/mongo-swift-driver/tree/main/Examples/VaporExample). 
```

Comment:
this is intentionally duplicative of top section to increase the chance people see it 🙂